docs/deploy.md

```diff
@@ -135,3 +135,51 @@ frontend:
 | /<*>.md        | /<*>.md        | 200 (Rewrite) |
 | /<*>.png       | /<*>.png       | 200 (Rewrite) |
 | /<*>           | /index.html    | 200 (Rewrite) |        
+
+
+## Docker
+
+>There is a simple demo for building docsify in docker.
```

Comment:
Can we remove this? I'm not sure why this is here (or why it's formatted using a blockquote)

Reply:
yep, it is unnecessary.